plone/portlet/collection/collection.py

```diff
@@ -224,6 +256,32 @@ def include_empty_footer(self):
         """
         return True
 
+    @memoize
+    def thumb_size(self):
+        ''' use  overrride value or read thumb_size from registry
+            image sizes must fit to value in allowed image sizes
+            none will suppress thumb!
+        '''
+        if self.data.ov_thumbsize > ' ':
+            return  self.data.ov_thumbsize
+        registry = getUtility(IRegistry)
+        settings = registry.forInterface(
+            ISiteSchema, prefix="plone", check=False)
+        if settings.no_thumbs_portlet:
+            return 'none'         
+        thumb_size_portlet = settings.thumb_size_portlet
+        return thumb_size_portlet
+
+    def getMimeTypeIcon(self,obj):     
+        fileo = obj.getObject().file
+        portal_url = api.portal.get().absolute_url()
```

Comment:
this comment is wrong, use https://github.com/plone/plone.app.layout/blob/master/plone/app/layout/navigation/root.py#L11